README.md

```diff
@@ -726,10 +727,10 @@ To upload an arbitrary number of files or folders, pass a space-separated list o
 
 ```shell
 # Specifying a mixed set of file and folder paths
-yarn ardrive upload-file -w wallet.json -F "${PUBLIC_FOLDER_ID}" --local-paths ./image.png ~/backups/ ../another_file.txt
+ardrive upload-file -w wallet.json -F "${PUBLIC_FOLDER_ID}" --local-paths ./image.png ~/backups/ ../another_file.txt
```

Comment:
Good catch

Reply:
In all the other examples in this doc, we've just been using mock IDs rather than variable placeholders to try and prevent confusion for users about what real values would look like. Can you please update all references like:
`${PUBLIC_FOLDER_ID}` to something like `6939b9e0-cc98-42cb-bae0-5888eca78885`?

Reply:
See here: 69f7974b697ba336c5c6752fb12cf0d8ae1a3261